src/pkg/boltz.ts

```diff
@@ -232,6 +233,9 @@ export class Boltz implements BoltzInterface {
         ]),
       };
     });
+
+    finalizer.finalize();
```

Comment:
I dont think this is needed as you manually finalizing the input 0th.

Please log the raw hex of the transaction, and decode it with element-cli to check if it works.